### DISCUSSION TITLE
[Bug] Order without address

### DISCUSSION THREAD

**Discussion by josias-hosang** [swellstores/community#443] [Help] [answered]
Yesterday a order in live environment was successfully placed (inclusive payment) but no shipping (nor billing) address is associated with this order: https://merz.swell.store/admin/orders/6554dbc5d457e10012dbd45a The email address was used before already but with no account creation and also no address. There are just abandoned carts (one of them with a address). Maybe this strange behaviour has something to do with your latest update covering address management? But this is worst case, because order cannot be shipped without address. Can you analyse this please and maybe provide some more information relating this order?

**Comment by logeshswell** [comment, marked as answer]
Associated cart for your order: https://merz.swell.store/admin/carts/6554db731df9e00012332ae3
Associated customer for your cart: https://merz.swell.store/admin/customers/65378048074be5001221ddf2

The above customer and the cart doesn't contain shipping or billing addresses associated with it. Hence, the converted order doesn't contain either address. 

To update the address for an account:
https://developers.swell.is/frontend-api/address-management#update-an-address

To update the address for a cart:
https://developers.swell.is/frontend-api/carts#update-cart-shipping-info
https://developers.swell.is/frontend-api/carts#update-cart-billing-info

**Reply by logeshswell** [reply]
I assumed that you were using custom storefront/checkout to end up with this issue.
But, it is impossible to place an order without shipping/billing address in Swell-hosted checkout irrespective of payment methods.

Could you please help me with a screenshot or recording to reproduce the issue?

**Reply by josias-hosang** [reply]
In the order reported above, I added an address manually afterwards via backend to fulfill the order. But meanwhile an other order was placed with no address (and no customer name) associated: https://merz.swell.store/admin/orders/657c8f02b586cd0012e3b2ed
I didn't change the order yet. Customer says, she provided an address successfully but it was not saved. She was also able to pay and complete the order.

Unfortunately we have no more information since this is your system and we have no developer or debug possibilities and I have no clue how this could happen. As far as I can see it from our site before checkout, customer visited with the following user agent string:
`"Mozilla/5.0 (iPhone; CPU iPhone OS 17_0_3 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/17.0.1 Mobile/15E148 Safari/604.1"`

There is also a ticket going on concerning this issue in merz.swell.store account.

**Reply by logeshswell** [reply]
Can you try one more time with the instructions provided by the customer to reproduce the issue, so that it will help us propose a fix for it faster, as we couldn't reproduce it in my Swell store.

**Comment by josias-hosang** [comment]
This would be hard, because these are not our customers. We provide the
shop software for our customer Merz.

Today there was again another order with same issue: no address provided.
Maybe you can get there more information?
Swell <https://merz.swell.store/admin/orders/100050>
merz.swell.store <https://merz.swell.store/admin/orders/100050>
[image: favicon.png] <https://merz.swell.store/admin/orders/100050>
<https://merz.swell.store/admin/orders/100050>


Am 27.12.2023 um 12:05 schrieb Logesh Dinakaran ***@***.***>:

﻿

Can you try one more time with the instructions provided by the customer to
reproduce the issue, so that it will help us propose a fix for it faster,
as we couldn't reproduce it in my Swell store.

—
Reply to this email directly, view it on GitHub
<https://github.com/orgs/swellstores/discussions/443#discussioncomment-7955395>,
or unsubscribe
<https://github.com/notifications/unsubscribe-auth/ALPY5SS74YJNQI44HC2SIJTYLP6I5AVCNFSM6AAAAAA7NWUKLGVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TSNJVGM4TK>
.
You are receiving this because you authored the thread.Message ID:
***@***.***>
